lib/src/main/java/com/diffplug/spotless/npm/NodeJSWrapper.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright 2016 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.diffplug.spotless.npm;
+
+import java.io.File;
+import java.util.Map;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+class NodeJSWrapper extends ReflectiveObjectWrapper {
+
+	public static final String V8_RUNTIME_CLASS = "com.eclipsesource.v8.V8";
+	public static final String V8_VALUE_CLASS = "com.eclipsesource.v8.V8Value";
+
+	public static final String WRAPPED_CLASS = "com.eclipsesource.v8.NodeJS";
+
+	private static final AtomicBoolean flagsSet = new AtomicBoolean(false);
+
+	public NodeJSWrapper(ClassLoader classLoader) {
+		super(Reflective.withClassLoader(classLoader),
+				reflective -> {
+					final boolean firstRun = flagsSet.compareAndSet(false, true);
+					if (firstRun) {
+						reflective.invokeStaticMethod(V8_RUNTIME_CLASS, "setFlags", "-color=false"); // required to run prettier on windows
```

Comment:
When running in Linux it outputs:
```
Error: unrecognized flag -color=false
Try --help for options
```
expected?

Reply:
it also does that on windows, but even though it prints this output: prettier accesses the flag later and adapts behavior accordingly. 🤷‍♂️

Reply:
OK.